crypto/stack/stack.c

```diff
@@ -118,32 +118,59 @@ OPENSSL_STACK *OPENSSL_sk_new(OPENSSL_sk_compfunc c)
     return (NULL);
 }
 
-int OPENSSL_sk_insert(OPENSSL_STACK *st, const void *data, int loc)
+int OPENSSL_sk_reserve(OPENSSL_STACK *st, int n)
 {
-    if (st == NULL || st->num < 0 || st->num == INT_MAX) {
+    size_t target;
+    size_t num_alloc = st->num_alloc;
```

Comment:
This is redundant assignment. It asks for redundant question why. Well, initially it was just question, and then it turned redundant. But I wish it wasn't posed to start with. I mean remove redundant assignment.

Reply:
It's gone now.